.github/workflows/build.yml

```diff
@@ -66,11 +66,36 @@ jobs:
         run: echo -e "${GITHUB_REF_NAME}" > cmd/thyra-server/version.txt
       - name: Build binary for ${{ matrix.target }} on ${{ matrix.arch}}
         run: GOOS=${{ matrix.target }} GOARCH=${{ matrix.arch }} CGO_ENABLED=1 CC=${{ matrix.cc }} CXX=${{ matrix.cxx }} go build -o thyra-server_${{ matrix.target }}_${{ matrix.arch }} ./cmd/thyra-server/
+      - name: Build installer binary
+        if: ${{ matrix.target == 'windows' }}
+        uses: JackMcKew/pyinstaller-action-windows@main
+        with:
+          path: scripts
+      - name: Package installer folder into Windows executable file
+        if: ${{ matrix.target == 'windows' }}
+        run: |
+          curl -L https://www.7-zip.org/a/lzma2201.7z --output lzma2201.7z
+          p7zip -d lzma2201.7z
+          echo -e ';!@Install@!UTF-8!\n
```

Comment:
I think it will be easier if you create this config.txt file in the repo and commit it